### ISSUE TITLE
Add a guide for Heroku deployment

### ISSUE THREAD

**Issue by glaschenko** [jmix-framework/jmix-v1-v2-docs#377]
For Jmix, deployment is trivial and we can mostly refer to Heroku Quick Start: 
https://devcenter.heroku.com/articles/getting-started-with-gradle-on-heroku

However, there are 2 things to mention:
-specify Java version in "system.properties" file
-specify the correct Jar in "Procfile".
You can find more on how this is done in Mario's video:
https://forum.jmix.io/t/learning-jmix-through-real-world-app-video-series-rent-your-stuff/1026/8

Basically, system.propeties should look smth like that (for Java 17):
_java.runtime.version=17_

And Procfile:
_web: java -Dserver.port=$PORT $JAVA_OPTS -jar build/libs/myapp-0.0.1-SNAPSHOT.jar_
